pkg/feature/builder.go

```diff
@@ -79,7 +79,7 @@ func (fb *featureBuilder) Manifests(paths ...string) *featureBuilder {
 		var manifests []manifest
 
 		for _, path := range paths {
-			manifests, err = loadManifestsFrom(path)
+			manifests, err = loadManifestsFrom(embeddedFiles, path)
```

Comment:
I am not quite sure about this one from the pure API perspective. By hardcoding it to the global variable (`//go:embed`) we are locking it for any other sources like we made possible before.

What I would think about instead is to make `fsys fs.FS` a field on the `featureBuilder` struct, add extra method such as `func (fb *featureBuilder) ManifestSource(fsys fs.FS) *featureBuilder` which will allow us to set it (with godoc mentioning it's a root fs from which manifest paths are loaded). By default, in the `Load()` func, where we instantiate a `Feature`, we would set it to `embeddedFiles` and if `ManifestSource` is not called in the builder chain that would stay like that. Not a big change, but gives us flexibility. WDYT?

Reply:
[e072121](https://github.com/maistra/opendatahub-operator/pull/15/commits/e07212186ff3ccdbb797121f22e83737a62ae0d5) done here